test/unit/ngsiv2/ngsiService/subscriptions-test.js

```diff
@@ -75,14 +75,13 @@ describe('Subscription tests', function() {
         nock.cleanAll();
```

Comment:
I understand that after the changes in this PR, no usage of  `/v1/` URLs in CB should remain. It would be a good idea to check it, e.g. recursive grep in test/unit/ngsiv2 directory.

Reply:
Checked:

```
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-node-lib$ grep -Rin NGSI9 test/unit/ngsiv2/
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-node-lib$
```

```
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-node-lib$ grep -Rin v1 test/unit/ngsiv2/
a620381@a620381-VirtualBox:~/workspace/finext/iotagent-node-lib$
```

Reply:
Great!

NTC